source/android/adaptivecards/src/main/res/layout/activity_type_ahead_search.xml

```diff
@@ -0,0 +1,152 @@
+                    android:visibility="gone"
+                    android:layout_gravity="center_vertical"
+                    android:clickable="true"
+                    android:contentDescription="Cancel"
+                    android:focusable="true"
+                    android:backgroundTint="#FFFFFF"
+                    tools:src="@android:drawable/ic_menu_close_clear_cancel"
+                    tools:visibility="visible" />
+            </LinearLayout>
+
+        </com.google.android.material.appbar.AppBarLayout>
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/search_results_list"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:clipToPadding="false"
+            android:descendantFocusability="afterDescendants"
+            android:scrollbars="vertical"
+            app:adapter="@{viewModel.adaptor}"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+            app:layout_constraintTop_toBottomOf="@+id/appbar"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent" />
+
+        <ProgressBar
+            android:id="@+id/customOverlayView"
+            android:indeterminateTint="#000"
+            android:contentDescription="Loading Options"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:scaleType="center"
+            android:layout_marginTop="160dp"
+            android:layout_gravity="center"
+            android:visibility="gone"
+            tools:visibility="visible"
+            app:layout_constraintTop_toBottomOf="@+id/appbar"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent" />
+
+        <androidx.appcompat.widget.AppCompatImageView
+            android:id="@+id/error_image"
+            android:layout_width="96dp"
+            android:layout_height="96dp"
+            android:layout_gravity="center"
+            android:contentDescription=""
+            android:layout_marginTop="160dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginEnd="16dp"
+            app:layout_constraintTop_toBottomOf="@+id/appbar"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            android:visibility="gone"
+            tools:src="@android:drawable/stat_notify_error"
+            tools:visibility="visible"/>
+
+        <TextView
+            android:id="@+id/error_msg_text"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="10dp"
+            android:layout_marginEnd="16dp"
+            android:text="Something went wrong!"
```

Comment:
>Something went wrong! [](http://example.com/codeflow?start=26&length=21)

Is this string localizable? This may trigger an accesibility issue

Reply:
removed it since its not required